cmake/onnxruntime_unittests.cmake

```diff
@@ -623,6 +623,7 @@ if (onnxruntime_BUILD_SHARED_LIB)
           protobuf::libprotobuf
           DEPENDS ${all_dependencies}
   )
+  target_compile_definitions(onnxruntime_shared_lib_test PUBLIC "ONNX_NAMESPACE=onnx")
```

Comment:
Is it needed by anything?

Reply:
It's needed by test_tensor_loader.cc. If we don't want to change test_tensor_loader.cc, then we don't need this change.